### PR TITLE
add OMIMPS

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
@sabrinatoro when you have a chance, could you look at this? (no rush)

I want to add the [OMIMPS myopathy, congenital](https://omim.org/phenotypicSeries/PS117000), however, there is already a term in Mondo with the same name (MONDO:0019952 'congenital myopathy'), that is equivalent to [Orphanet](https://www.orpha.net/consor/www/cgi-bin/OC_Exp.php?Expert=97245).

I don't think the Orphanet and OMIM classes are equivalent. The orphanet class has [36 genes involved](https://www.orpha.net/consor/www/cgi-bin/Disease_Genes_Simple.php?lng=EN&LnkId=12867&Typ=Pat&diseaseType=Gen&from=rightMenu), whereas, the [OMIMPS](https://omim.org/phenotypicSeries/PS117000) has much fewer.

My proposed approach is to name the OMIM class 'myopathy, familial congenital' (or it could be myopathy, hereditary, congenital). I added the equivalence axiom:

```
'congenital myopathy'
 and ('bearer of' some inherited)
````

and it is inferred to be a child of the Orphanet class, which makes sense to me.

Could you review this and make sure this approach makes sense?

This is a work in progress, there are other subclasses in the OMIMPS that need to be added as part of the [addition of new OMIM terms](https://docs.google.com/spreadsheets/d/164M99rbBZ5W3qTdhTk_egBaWdFX0rMnm2l1oMIRuMls/edit#gid=1984174296).

address https://github.com/monarch-initiative/mondo/issues/6133

### CODE REVIEW

**Comment by sabrinatoro** [comment]
@nicolevasilevsky This is tricky...
Why do you think OMIMPS:117000 and Orphanet:97245 are different? 

1) OMIMPS says: "Myopathy, congenital (see also nemaline myopathy (PS161800), myofibrillar myopathy (PS601419), and centronuclear myopathy (PS160150).
Orphanet is a parent term grouping for Centronuclear myopathy, Congenital myopathy with cores, Congenital nemaline myopathy"
It looks like both terms group at least some of the same types of myopathy.

2) If we say that "myopathy, hereditary, congenital" = 'congenital myopathy' and ('bearer of' some inherited)
Does it mean that some congenital myopathy are not inherited... Is that true?

3) I see other "congenital myopathy" terms that might have to be reclassified under "hereditary" (e.g MONDO:0013890) 

In summary: can we confirm that some congenital myopathies are not hereditary? (maybe we need to ask experts) 
(I hope this is helpful)

**Comment by nicolevasilevsky** [comment]
thanks @sabrinatoro! this help!

**Comment by nicolevasilevsky** [comment]
I think it is inherited: https://rarediseases.org/rare-diseases/myopathy-congenital-batten-turner-type/
